CODEOWNERS: assign operator/pkg/{gateway-api,model} to @cilium/sig-servicemesh

tklauser opened the pull request:
Members of the @cilium/operator team might not have enough context on servicemesh specifics, so request a review from @cilium/sig-servicemesh as well.

sayboras reviewed (approved):
Thanks :100: :heavy_check_mark:

tklauser committed:
CODEOWNERS: assign operator/pkg/{gateway-api,model} to sig-servicemesh team

Members of the @cilium/operator team might not have enough context on
servicemesh specifics, so request a review from @cilium/sig-servicemesh
as well.

Signed-off-by: Tobias Klauser <tobias@cilium.io>
